book/src/in-depth-guides/tabs/graph.md

```diff
@@ -0,0 +1,83 @@
+# Microsoft Graph client
+
+The client App exposes a `graph` property that gives type-safe access to  Microsoft Graph functions. When graph functions are invoked, the app attaches an MSAL bearer token to the request so that the call can be authenticated and authorized. 
```

Comment:
Nice, we're moving the other `App` to follow suit: https://github.com/microsoft/teams.ts/pull/202